File: oscexchange/src/main/java/ac/at/fhstp/digitech/oscexchange/OSCRawReceiveListener.java
package ac.at.fhstp.digitech.oscexchange;

/**
 * Listener for when OSCArgs were received
 */
@FunctionalInterface
public interface OSCRawReceiveListener {

    /**
     * Handles the received OSCArgs
     *
     * @param args The received args
     */
    @PublicApi
    void handle(OSCArgs args);

}

File: oscexchange/src/main/java/ac/at/fhstp/digitech/oscexchange/OSCRawReceiveListener.kt
package ac.at.fhstp.digitech.oscexchange

/**
 * Listener for when OSCArgs were received
 */
fun interface OSCRawReceiveListener {
    /**
     * Handles the received OSCArgs
     *
     * @param args The received args
     */
    @PublicApi
    fun handle(args: OSCArgs)
}